From 430e49c16d3dc024c5ae05798853656ed479702b Mon Sep 17 00:00:00 2001
From: Zhantao Tang <zhantao.tang@windriver.com>
Date: Tue, 24 Dec 2019 02:18:28 +0000
Subject: [PATCH 239/245] Revert "arm64: dts: ls1088a: Add USB support"

This reverts commit a15e29a4c8ab92e2278fa5348882664822f72aa8.
There already exist usb3@3100000 and usb3@3110000 nodes, duplicated
ones should be deleted.

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 .../boot/dts/freescale/fsl-ls1088a-rdb.dts    |  8 -------
 .../arm64/boot/dts/freescale/fsl-ls1088a.dtsi | 24 -------------------
 2 files changed, 32 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1088a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-ls1088a-rdb.dts
index 9cadf27fbc82..9b98dfe54211 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1088a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1088a-rdb.dts
@@ -107,14 +107,6 @@
 	status = "okay";
 };
 
-&usb0 {
-	status = "okay";
-};
-
-&usb1 {
-	status = "okay";
-};
-
 &esdhc {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
index ed3cedd914f9..2a96cd37734a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
@@ -515,26 +515,6 @@
 			status = "disabled";
 		};
 
-		usb0: usb3@3100000 {
-			compatible = "snps,dwc3";
-			reg = <0x0 0x3100000 0x0 0x10000>;
-			interrupts = <0 80 IRQ_TYPE_LEVEL_HIGH>;
-			dr_mode = "host";
-			snps,quirk-frame-length-adjustment = <0x20>;
-			snps,dis_rxdet_inp3_quirk;
-			status = "disabled";
-		};
-
-		usb1: usb3@3110000 {
-			compatible = "snps,dwc3";
-			reg = <0x0 0x3110000 0x0 0x10000>;
-			interrupts = <0 81 IRQ_TYPE_LEVEL_HIGH>;
-			dr_mode = "host";
-			snps,quirk-frame-length-adjustment = <0x20>;
-			snps,dis_rxdet_inp3_quirk;
-			status = "disabled";
-		};
-
 		usb0: usb3@3100000 {
 			compatible = "snps,dwc3";
 			reg = <0x0 0x3100000 0x0 0x10000>;
@@ -542,8 +522,6 @@
 			dr_mode = "host";
 			configure-gfladj;
 			snps,dis_rxdet_inp3_quirk;
-			snps,incr-burst-type-adjustment = <1>, <4>, <8>, <16>;
-			snps,host-vbus-glitches;
 		};
 
 		usb1: usb3@3110000 {
@@ -553,8 +531,6 @@
 			dr_mode = "host";
 			configure-gfladj;
 			snps,dis_rxdet_inp3_quirk;
-			snps,incr-burst-type-adjustment = <1>, <4>, <8>, <16>;
-			snps,host-vbus-glitches;
 		};
 
 		sata: sata@3200000 {
-- 
2.17.1

